gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Cell.java

```diff
@@ -971,8 +971,8 @@ void merge (Cell cell) {
 		if (cell.expandX != null) expandX = cell.expandX;
 		if (cell.expandY != null) expandY = cell.expandY;
 		if (cell.colspan != null) colspan = cell.colspan;
-		if (cell.uniformX != null) uniformX = cell.uniformX;
-		if (cell.uniformY != null) uniformY = cell.uniformY;
```

Comment:
This is not equivalent and is a reason why `Boolean` is used instead of `boolean`. What crashes are you experiencing? None of the changed lines could cause a crash, regardless of the value in `uniformX/Y`.